src/main/asciidoc/java/index.adoc

```diff
@@ -362,9 +362,10 @@ with respect to concurrency, scaling and deployment.
 
 To use this model, you write your code as set of *verticles*.
 
-Verticles are chunks of code that get deployed and
-run by Vert.x. Verticles can be written in any of the languages that Vert.x supports and a single application
-can include verticles written in multiple languages.
+Verticles are chunks of code that get deployed and run by Vert.x. Vert.x will create N event loops threads
```

Comment:
I would be clearer to say "A Vert.x instance maintains N event loop threads (where N by default is core*2) by default"